### PR TITLE
fix: working fairplay check

### PR DESCRIPTION
**Pull request by slowmove**
Fairplay check is broken in master, since `initDataTypes` is required
+ simple demo page

### CODE REVIEW

**Comment by martinstark** [comment]
repo uses yarn (yarn.lock), I'd suggest removing the package-lock file from the pr, otherwise it looks good

**Comment by martinstark** [comment]
adding parcel as a second javascript compiler when there already is a webpack setup seems a bit overkill, could use webpack dev server to run the demo page instead?

File: package.json [hunk 1]
```diff
@@ -52,6 +56,7 @@
     "eslint_d": "11.1.1",
     "husky": "7.0.4",
     "lint-staged": "11.1.2",
+    "parcel": "^2.4.0",
```
Comment (martinstark) [inline package.json:59]: adding parcel as a second javascript compiler when there already is a webpack setup seems a bit overkill, could use webpack dev server to run the demo page instead?

https://webpack.js.org/configuration/dev-server/

(alternatively swap webpack for parcel completely, by updating the automated release script actions and removing webpack)

**Review by martinstark — COMMENTED** [review]
repo uses yarn (yarn.lock), please remove package-lock from pr

File: package.json (hunk 1 above)
Comment (martinstark) [reply package.json:59]: (to get the fairplay fix merged quickly I'd suggest opening a separate PR with just the fairplay fix, the demo page could then be a separate `feat:` commit for the automated release notes)

**Comment by slowmove** [comment]
Don't have time for this.

**Pull request closed without merging**